src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-cairocffi.py

```diff
@@ -0,0 +1,23 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2021 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+from ctypes.util import find_library
+
+from PyInstaller.utils.hooks import collect_data_files
+
+datas = collect_data_files('cairocffi')
+
+libs = ['cairo-2', 'cairo', 'libcairo-2']
+binaries = []
+for lib in libs:
+    p = find_library(lib)
+    if p is not None:
+        binaries += [(find_library(lib), '.')]
```

Comment:
`ctypes.util.find_library` returns only library name without path on some OSes (linux). So that will not work there.

I think we usually handle that by using  `PyInstaller.depend.utils._resolveCtypesImports`, like [here](https://github.com/pyinstaller/pyinstaller-hooks-contrib/blob/1cbfd2c1c0e9fda5a4fd39d657a6df95c2028ca9/src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-usb.py#L60-L89) or [here](https://github.com/pyinstaller/pyinstaller-hooks-contrib/blob/1cbfd2c1c0e9fda5a4fd39d657a6df95c2028ca9/src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pymediainfo.py#L21-L30).

Reply:
Done in https://github.com/pyinstaller/pyinstaller-hooks-contrib/pull/347/commits/1e2e563a127a8f60cccda74ace02f3b19b05810f